### PR TITLE
Added plot directive to CVScores docs

### PR DESCRIPTION
**Pull request by pdamodaran**
The following changes were made to address #687:

1. deleted the Python script previously used to generate the images
2. deleted the original image files
3. updated the .rst file with the new plot directive, including adding the :context: close-figs and :alt: options

Notes
------
Of the two new images produced by the plot directive, the second image does not match its original

### CODE REVIEW

**Comment by pdamodaran** [comment]
Updated Second image 

cross_validation-2

**Comment by pdamodaran** [comment]
Original Second Image

cv_scores_regressor

File: docs/api/model_selection/cross_validation.rst [hunk 1]
```diff
@@ -19,26 +19,21 @@ Classification
 
 In the following example we show how to visualize cross-validated scores for a classification model. After loading a ``DataFrame``, we create a ``StratifiedKFold`` cross-validation strategy to ensure all of our classes in each split are represented with the same proportion. We then fit the ``CVScores`` visualizer using the ``f1_weighted`` scoring metric as opposed to the default metric, accuracy, to get a better sense of the relationship of precision and recall in our classifier across all of our folds.
```
[the remark this answers is not in the document]
Comment (Kautumn06) [reply docs/api/model_selection/cross_validation.rst]: Good catch @lwgray! I think updating this text in both places will help make the documentation more clear.

**Comment by pdamodaran** [comment]
I have made the changes as suggested by @lwgray and @Kautumn06.  Thanks for your feedback!

**Comment by pdamodaran** [comment]
@Kautumn06 - I added random_state=42 to the rst file

**Comment by Kautumn06** [comment]
Thank you @pdamodaran for working so hard to help us update our docs! I pushed some small changes to remove the extra spacing and add additional comments to the code blocks. Once the tests pass, I'll go ahead and merge it in. Thanks again!

**Pull request merged**